www/main/viewer/utils/chromecast.js

```diff
@@ -1,182 +1,192 @@
-const remote = require("@electron/remote");
+/* global chrome */
```

Comment:
Veerji, I'm not sure if I'm familiar with this one. Can you help me know this one?

Reply:
@theamanjs chrome object is imported after a script is initialized. If we don't specify this linter will give an undefined error, because the object is intialized on run time.
https://eslint.org/docs/latest/user-guide/configuring/language-options#specifying-globals

Reply:
Thankyou veerji. This is helpful :raised_hands: